filterable-file-log/serializer.lua

```diff
@@ -124,4 +133,20 @@ function whitelist_filter(t, whitelist)
   end, t)
 end
 
+function convert_error_message(msg)
+  local e
+  if string.find(msg, "No API key") then
```

Comment:
Do you know of the performance implications of `string.find` vs other pattern matching functions available?

Reply:
I hadn't looked in to this too deeply prior to this PR, but I did take a bit of time to update myself. I'll see if I can find another article or two to back up the idea behind this, just cause it's tough to base this off of a single article, but here's an interesting read (summary below).
https://www.cult-of-tech.net/2017/06/lua-string-compare-performance-testing-nginx-lua/
Looks like if you're doing an exact match, string.find performs very well. Once you start throwing in complex searches or wildcards, it begins having significant performance implications.

Reply:
Any particular pattern matching you had in mind that you may think would perform better? Glad to do a little digging.